### PR TITLE
Add ebpc support to atoumtask

### PR DESCRIPTION
**Pull request by oxman**
Add ebpc support to AtoumTask for Phing

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # `dev-master`
 
+* [#619](https://github.com/atoum/atoum/pull/619) Add ebpc support to AtoumTask for Phing ([@oxman])
```
Comment (jubianchi) [inline CHANGELOG.md]: "Add branches and paths coverage ..."

**Comment by jubianchi** [comment]
A minor change to the readme and we'll merge.

Thanks

**Comment by Hywan** [comment]
ping?

**Comment by shulard** [comment]
Hello I think it's a pity to block that PR just because a merge conflict... @oxman, are you ok if I make a PR on your branch ? Then if you accept it, it'll update the code here directly ? Or, if you prefer I can cherry pick your commits for another PR ?

**Comment by oxman** [comment]
You can update it directly :-)
I said i won't upgrade this PR to resolve conflict, because i got conflict because this PR make time to be approuved.

**Comment by shulard** [comment]
Yeah, I understand and I viewed that you don't want to update it... To work in it directly, I need write access to your branch, I think... Is there another way to do ?

**Comment by jubianchi** [comment]
@oxman @shulard don't know if you can update the PR so I can resolve the conflict on your branch. See https://github.com/blog/2247-improving-collaboration-with-forks

**Comment by shulard** [comment]
I've viewed that Github feature, seems the better solution because I haven't right access 😄

**Pull request closed without merging**